.github/workflows/selenium_tests.yml

```
@@ -0,0 +1,80 @@
+name: Selenium Tests
+
+on:
+  pull_request:
+    types: [labeled, synchronize, opened, reopened]
+
+permissions:
+   contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  selenium-sqlite:
+    if: contains(github.event.pull_request.labels.*.name, 'selenium')
+    runs-on: ubuntu-latest
+    name: Selenium tests - SQLite
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+          cache: 'pip'
+          cache-dependency-path: 'tests/requirements/py3.txt'
+
+      - name: Install and upgrade packaging tools
+        run: python -m pip install --upgrade pip setuptools wheel
+      - run: python -m pip install -r tests/requirements/py3.txt .
+
+      - name: Run selenium tests
+        working-directory: ./tests/
+        run: python -Wall runtests.py --verbosity 2 --noinput --selenium=chrome --headles --settings=test_sqlite --parallel 2
+
+  selenium-postgres:
+    if: contains(github.event.pull_request.labels.*.name, 'selenium')
+    runs-on: ubuntu-latest
+    name: Selenium tests - PostgreSQL
+    services:
+      postgres:
+        image: postgres
```

Maybe:
```suggestion
        image: postgres:12-alpine
```
to use the same version in all jobs.